### PR TITLE
Mail: Remove redundant body creation in case of only a preamble is set

### PR DESCRIPTION
**Pull request by nupplaphil**

### CODE REVIEW

File: src/Util/EMailer/SystemMailBuilder.php [hunk 1]
```diff
@@ -98,7 +94,7 @@ protected function getSubject()
 	 */
 	protected function getHtmlMessage()
 	{
-		$htmlVersion = BBCode::convert($this->body);
+		$htmlVersion = !empty($this->body) ? BBCode::convert($this->body) : '';
```
Comment (MrPetovan) [inline src/Util/EMailer/SystemMailBuilder.php]: Please default the `$body` parameter to empty string on line 72 to avoid this check.

**Comment by MrPetovan** [comment]
Committing in the future again, huh.

**Comment by nupplaphil** [comment]
Don't remind me .. I'm just waiting for my new laptop and to wipe this laptop with all the system on it .. ;-) .. I've to currently adjust the clock manually at every boot (NTP doesn't work, don't ask me why ^^)

**Comment by AlfredSK** [comment]
Well...

db-upgrade_20200307143131

The text is now completely missing. :-)

**Comment by AlfredSK** [comment]
The plaintext version is ok.

**Comment by nupplaphil** [comment]
Damn.. I'll fix it.

**Comment by tobiasd** [comment]
I would guess, Lord Sauron is simply too cool for HTML mails ;-)

**Pull request merged**